### DISCUSSION TITLE
Any plan to support systemc ?

### DISCUSSION THREAD

**Discussion by kevinyuan** [simbricks/simbricks#121] [General]
Dear simbricks dev team,

Do you have any plan to support systemc ?

**Comment by FreakyPenguin** [comment]
Hi @kevinyuan!
Excellent question! And this is actually something we are currently working on. Doing this in general seems to be quite manageable. Although I expect this will not be quite plug and play, as different models seem to have somewhat different ways of implementing their interfaces in the top-level module. Do you have a specific use-case in mind? Maybe some open-source systemC model we could also use for development? (NVDLA is currently a major one on our todo list).

Antoine

**Reply by cnjsdfcy** [reply]
Any progress on supporting SystemC? Thanks~

**Reply by q713** [reply]
Hi,
unfortunately we had a lot of other things on our todo list recently, including a re-write of our orchestration framework and we didn't have the time to implement support for SystemC.
That being said, we still plan to add support for SystemC in the future but we can't make a prediction on when we will have the time to do so.

Jakob

**Reply by jonas-kaufmann** [reply]
@cnjsdfcy Can you tell us a bit more about your use-case? As Antoine mentioned, there seem to be different types of model interfaces. Would be helpful to get an idea of what to focus on first. Do you have a specific model in mind?
